man/systemd.link.xml

```diff
@@ -359,6 +357,17 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>AutoNegotiation=</varname></term>
+        <listitem>
+          <para>The AutoNegotiation is an ethernet procedure by which two connected devices
+          choose common transmission parameters, such as speed, duplex mode, and flow control.
+          Takes a boolean value. Defaults to "unset".</para>
```

Comment:
"Unset by defaults, which means that the kernel default will be used."

Please add an empty line between paragraphs.